airflow/www/views.py

```diff
@@ -2490,6 +2491,7 @@ class ConnectionModelView(wwwutils.SuperUserMixin, AirflowModelView):
         'extra__jdbc__drv_clsname': StringField('Driver Class'),
         'extra__google_cloud_platform__project': StringField('Project Id'),
         'extra__google_cloud_platform__key_path': StringField('Keyfile Path'),
+        'extra__google_cloud_platform__key_json': StringField('Key JSON'),
```

Comment:
nit: for consistency with ServiceAccountCredentials class, maybe call it `Keyfile Json` or `Keyfile Dict`? :)

Reply:
Makes sense. I'll change it to `keyfile_dict`.